gmr/sklearn.py

```diff
@@ -52,31 +65,30 @@ def __init__(self, n_components, priors=None, means=None, covariances=None,
         self.R_diff = R_diff
         self.n_iter = n_iter
         self.init_params = init_params
-    
-    def fit(self, X, y):
-        self.gmm = GMM(self.n_components, priors=self.priors, means=self.means, 
+
+        self.gmm_ = GMM(self.n_components, priors=self.priors, means=self.means,
```

Comment:
sklearn-compatible estimators shouldn't do any fancy initialization in their constructors. This line should be moved to the fit function.